corehq/apps/app_manager/templates/app_manager/partials/forms/case_config_ko_templates.html

```diff
@@ -234,6 +234,7 @@ <h4 class="panel-title panel-title-nolink">
           <!-- ko if: $data.type && $data.type() == "user-property" -->
             {% trans "User Case Property" %}
             <span
+                  style="position: absolute; margin-top: -2px"
```

Comment:
try with position relative?

Reply:
Didn't seem to work, so leaving it as is for now